kasa/modules/usage.py

```diff
@@ -50,22 +51,53 @@ def usage_this_month(self):
 
         return converted.pop()
 
-    async def get_daystat(self, *, year=None, month=None):
-        """Return daily stats for the given year & month."""
+    async def get_raw_daystat(self, *, year=None, month=None) -> Dict:
+        """Return raw daily stats for the given year & month."""
         if year is None:
             year = datetime.now().year
         if month is None:
             month = datetime.now().month
 
         return await self.call("get_daystat", {"year": year, "month": month})
 
-    async def get_monthstat(self, *, year=None):
-        """Return monthly stats for the given year."""
+    async def get_raw_monthstat(self, *, year=None) -> Dict:
+        """Return raw monthly stats for the given year."""
         if year is None:
             year = datetime.now().year
 
         return await self.call("get_monthstat", {"year": year})
 
+    async def get_daystat(self, *, year=None, month=None) -> Dict:
+        """Return daily stats for the given year & month as a dictionary of {day: time, ...}."""
+        data = await self.get_raw_daystat(year=year, month=month)
+        data = self._convert_stat_data(data["day_list"], entry_key="day")
+        return data
+
+    async def get_monthstat(self, *, year=None) -> Dict:
+        """Return monthly stats for the given year as a dictionary of {month: time, ...}."""
+        data = await self.get_raw_monthstat(year=year)
+        data = self._convert_stat_data(data["month_list"], entry_key="month")
+        return data
+
     async def erase_stats(self):
         """Erase all stats."""
         return await self.call("erase_runtime_stat")
+
+    def _convert_stat_data(self, data, entry_key) -> Dict:
+        """Return usage information keyed with the day/month.
+
+        The incoming data is a list of dictionaries:
+               [{'year':      int,
```

Comment:
```suggestion
        The incoming data is a list of dictionaries::

               [{'year':      int,
```
Using a double `::` and an empty before the example data outputs the example in the docs in nicer format. For testing, you can compile the docs locally using `make html` inside the `docs` directory.

Reply:
Also, maybe it's better to document the format in the `get_raw_data` methods?